src/Network/Network/help/Update-AzVirtualRouterPeer.md

```diff
@@ -0,0 +1,263 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version:https://docs.microsoft.com/en-us/powershell/module/az.network/update-azvirtualrouterpeer
```

Comment:
Not sure how docs would handle this, please add space between "online version" and URI. Same applies to other help files

Reply:
Done